src/test/java/org/candlepin/subscriptions/resource/CapacityResourceTest.java

```diff
@@ -559,7 +559,7 @@ void testReportByMetricIdShouldCalculateCapacityVirtualSockets() {
             null);
 
     CapacitySnapshotByMetricId capacitySnapshot = report.getData().get(0);
-    assertEquals(5, capacitySnapshot.getValue());
+    assertEquals(2, capacitySnapshot.getValue());
```

Comment:
@kahowell @kflahert I would like your opinions on the changes I had to make here and further down in this unit test.  By coercing the `ReportCategory` into a `HypervisorReportCategory` at the start of `getCapacityReportByMetricId`, the meaning of these tests changed: now instead of the `VIRTUAL` report category returning the 5 sockets associated with `hypervisorSockets`, it returns the 2 sockets associated with standard `sockets`.  I believe this is the correct behavior but would like a double-check.

I'm also on the fence about whether we should even be accepting `CLOUD`, `VIRTUAL`, and `PHYSICAL` as capacity report categories given there is no meaningful distinction between those categories for capacity.  Having them preserves symmetry with tally, but I am thinking we should just separate into `CapacityReportCategory` and `TallyReportCategory`

Reply:
These test updates make sense to me

Reply:
:+1: